src/modules/tests/__tests__/tests.selector.spec.ts

```diff
@@ -156,8 +156,8 @@ describe('testsSelector', () => {
     });
     it('should return mes-red class for non listed activity code 86', () => {
       testState.activityCode = '86';
-      const result = getTestOutcome(testState);
-      expect(result).toBe('Terminated');
+      const result = getTestOutcomeClass(testState);
+      expect(result).toBe('mes-red');
     });
   });
 
```

Comment:
Unit test for `getShowMeQuestionOptions`?

Reply:
oh yes. good spot. will do that now.